### PR TITLE
Remove inactive reviewers

### LINKED ISSUE #13702: Consider removing reviewers who haven't reviewed in over a year
We currently have 63 reviewers who have had a review requested of them on a PR created in the past year but who haven't reviewed anything; some of these looking quickly are people who no longer work on the web platform.

If we want to look at new ways of assigning reviews, we should probably start looking into trimming the META.yml lists.

### PR DESCRIPTION
**Pull request by gsnedders**
This removes reviewers who have had more than three review requests in the past year but who haven't reviewed anything. Fixes #13702.

Places where this leaves us with no reviewers:

* css/css-display/run-in (but parent has reviewers, so let's just kill that)
* ~css/css-position (very few PRs; @atanassov is the current reviewer)~
* ~css/css-scrollbars (very few PRs; @tantek is current? maybe +@upsuper?)~
* encrypted-media (very few PRs; @ddorwin is current)
* ~feature-policy (very few PRs; @clelland is current)~
* ~gamepad (very few PRs; @luser is current)~

I believe everyone removed here was pinged in https://github.com/web-platform-tests/wpt/issues/13702#issuecomment-433376601, so I don't think we need to check about anyone except for the above. If the people mentioned above want to remain reviewers, please speak now.

### CODE REVIEW

**Review by astearns — APPROVED** [review]

**Review by marcoscaceres — APPROVED** [review]
Approved for Payment method id.

**Comment by marcoscaceres** [comment]
> gamepad (very few PRs; @luser is current)

Happy to take gamepad as reviewer. I'll add myself.

**Comment by marcoscaceres** [comment]
@bakulf might also be good for feature-policy?

**Comment by upsuper** [comment]
I'm fine for adding me.

**Comment by upsuper** [comment]
I'm fine for adding me to css-scrollbars.

**Comment by bakulf** [comment]
I'm fine to be added as feature-policy reviewer.

File: feature-policy/META.yml [hunk 1]
```diff
@@ -1,3 +1,2 @@
 spec: https://wicg.github.io/feature-policy/
 suggested_reviewers:
```
Comment (marcoscaceres) [inline feature-policy/META.yml]: As confirmed in the main comments, you can apply the following suggestion: 
 
```suggestion
 suggested_reviewers:
  - bakulf 
```

**Comment by marcoscaceres** [comment]
Thanks @bakulf. Suggested you https://github.com/web-platform-tests/wpt/pull/14798/files#r247437966

**Comment by gsnedders** [comment]
OK, so that leaves `css/css-position` and `encrypted-media` without reviewers.

**Comment by astearns** [comment]
You can add me as the css-position reviewer for now.

**Comment by gsnedders** [comment]
For encrypted-media: @ddorwin, @jrummell-chromium? @cpearce? @jyavenard?

**Comment by mounirlamouri** [comment]
> For encrypted-media: @ddorwin, @jrummell-chromium? @cpearce? @jyavenard?

Could you swap @ddorwine with @joeyparrish ?

**Review by birtles — APPROVED** [review]

**Review by alvestrand — APPROVED** [review]
LGTM for mediacapture-main/ and webrtc/

**Review by cbiesinger — APPROVED** [review]
lgtm for css-flexbox

**Pull request merged**